css/index.css

```diff
@@ -1,3 +1,476 @@
+	margin-top: 76px;
+	padding-top: 39px;
+  }
+  
+  .main-content .top-content {
+	padding: 0 10px;
+  }
+  @media(max-width: 800px){
+     .main-content .top-content{
+         display: flex;        
+     }
+  }
+  @media(max-width: 500px){
+    .main-content .top-content{
+      flex-direction: column;
+      padding-left: 0;
+    }
+  }
+ 
+  
+  .main-content .top-content .text-content {
+	width: 410px;
+	display: inline-block;
+	margin-right: 34px;
+  }
+  @media(max-width: 500px){ 
+    .main-content .top-content .text-content{ 
+        width: 100%; 
+    } 
+  }
+  
+  .main-content .top-content .text-content:last-child {
+	margin-right: 0;
+  }
+  @media(max-width: 500px){
+    .main-content .top-content .text-content:last-child {
+        margin: 30px 0 0 0;
+    }
+  }
+  
+  .main-content .middle-img {
+	margin-top: 45px;
+  }
+  @media(max-width: 800px){
+    .main-content .middle-img{
+        max-width: 100%;
+        height: auto;
+    }
+  }
+  @media(max-width: 500px){
+      .main-content .middle-img{
+          height: auto;
+      }
+  }
+  
+  .main-content .bottom-content {
+	margin-top: 37px;
+	padding: 0 10px;
+  }
+  @media(max-width: 800px){
+    .main-content .bottom-content{
+        display: flex;
+    }
+  }
+  @media(max-width: 500px){
+    .main-content .bottom-content{
+        flex-direction: column;
+        align-items: stretch;
+        justify-content: space-between;
+        padding: 0;
+        height: 420px;
+    }
+  }
+ 
+  
+  .main-content .bottom-content .text-content {
+	width: 275px;
+	display: inline-block;
+	margin-right: 12px;
+  }
+   @media(max-width: 500px){ 
+     .main-content .bottom-content .text-content{ 
+         width: 100%; 
+     } 
+  } 
+ 
+  
+  .main-content .bottom-content .text-content:last-child {
+	margin-right: 0;
+  }
+  
+  .contact {
+	margin-top: 57px;
+	border-top: 1px solid black;
+	padding: 39px 10px 0;
+  }
+  
+  .contact p {
+	margin: 20px 0;
+  }
+  
+  footer {
+	margin: 32px 0;
+	text-align: center;
+	font-size: 14px;
+  }
+
+  
+
+  /* services */
+
+ .services .services-header-img {
+	  display: block;
+	margin: 30px 0;
+  } 
+  @media (max-width: 800px) {
+    .services .services-header-img {
+        max-width: 100%;
+        height: auto;
+    }
+}
+.services .main-info {
+	border-bottom: 1px solid black;
+	padding-bottom: 25px;
+
+}
+@media (max-width: 500px){
+    .services .main-info{
+        display: flex;
+        flex-flow: column wrap;
+        align-items: center;
+    }
+}
+.services h1 {
+	font-size: 25px ; 
+}
+	
+.services .main-contents{
+		border-bottom: 1px solid black;
+		display: flex;
+		flex-flow: row wrap;
+	  justify-content: space-between;
+		padding-top: 25px;
+		height: 800px;
+}
+@media (max-width: 800px) {
+    .services .main-contents{
+        height: 1100px
+    }
+}
+@media (max-width: 500px){
+    .services .main-contents{
+        flex-flow: column nowrap;
+        height: auto;
+        align-items: stretch;
+    }
```

Comment:
Keep in mind that you're able to put multiple CSS selectors in one media query; this style is still viable (and makes sense to me, as you're keeping the styles ordered), but it is a good bit of repetition with the many media queries.
```css
@media (max-width: 500px) {
    .something {
        display: flex;
    }
    .something-else {
        padding: 20px;
    }
}